manifest_test.go

```diff
@@ -119,3 +120,74 @@ func TestReadManifestErrors(t *testing.T) {
 		}
 	}
 }
+
+func TestValidateManifest(t *testing.T) {
+	cases := []struct {
+		tomlString string
+		want       []error
+	}{
+		{
+			tomlString: `
+			[[dependencies]]
+        name = "github.com/foo/bar"
```

Comment:
can we also validate that there aren't any extra fields within `[[dependencies]]` or `[[overrides]]` items?